src/iosrv/srv.c

```diff
@@ -257,7 +256,10 @@ check_sleep_list()
 	if (dss_xstream_exiting(dx))
 		shutdown = true;
 
-	daos_gettime_coarse(&now);
+	if (d_list_empty(&dx->dx_sleep_ult_list))
+		return;
+
+	now = daos_get_ntime();
```

Comment:
It's on performance critical path, CLOCK_MONOTONIC_COARSE is supposed to be faster than CLOCK_MONOTONIC, please change it back. (use coarse aversion get time function).

Reply:
ah, I missed that. Xuezhao also mentioned that. but I missed COARSE when I checked. I will fix that.  thanks.